src/app/static/src/app/components/outputTable/TableReshapeData.vue

```diff
@@ -0,0 +1,92 @@
+        const filterDataIdtoId = computed(() => {
+            const filters: Filter[] = store.getters["modelOutput/tableFilters"] || [];
+            const dataIdToId: Record<string, string> = {};
+            filters.forEach(f => {
+                dataIdToId[f.column_id] = f.id;
+            });
+            return dataIdToId;
+        });
+        
+        const presetMetadata = computed(() => {
+            const currentPreset = store.state.plottingSelections.table.preset;
+            const presetMetadata = store.state.modelCalibrate.metadata?.tableMetadata.presets;
+            const metadata = presetMetadata?.find(m => m.defaults.id === currentPreset);
+            if (currentPreset && presetMetadata && metadata) {
+                return { row: metadata.defaults.row, column: metadata.defaults.column };
+            }
+            return null;
+        });
+
+        const reshapedData = computed(() => {
+            const data = props.data;
+            if (presetMetadata.value && data) {
+                const reshapedData: any[] = [];
+                const rowKey = presetMetadata.value.row;
+                let rowOptions: FilterOption[] = [];
+                if (rowKey === "area_id") {
+                    if (features.value) {
+                        rowOptions = features.value.map(f => {
+                            return { id: f.properties.area_id, label: f.properties.area_name };
+                        });
+                    }
+                } else {
+                    rowOptions = selections.value.selectedFilterOptions[filterDataIdtoId.value[rowKey]];
+                }
+                const columnKey = presetMetadata.value.column;
+                for (let i = 0; i < data.length; i++) {
+                    const currentRow = data[i];
+                    const rowVal = currentRow[rowKey];
+                    const rowLabel = rowOptions.find(op => op.id === rowVal)?.label;
+                    const index = reshapedData.findIndex(d => d.label === rowLabel);
+                    if (index === -1) {
+                        reshapedData.push({
+                            "label": rowLabel,
+                            [`mean_${currentRow[columnKey]}`]: currentRow.mean,
+                            [`upper_${currentRow[columnKey]}`]: currentRow.upper,
+                            [`lower_${currentRow[columnKey]}`]: currentRow.lower,
+                        })
+                    } else {
+                        reshapedData[index][`mean_${currentRow[columnKey]}`] = currentRow.mean;
+                        reshapedData[index][`upper_${currentRow[columnKey]}`] = currentRow.upper;
+                        reshapedData[index][`lower_${currentRow[columnKey]}`] = currentRow.lower;
+                    }
+                }
+                return reshapedData;
+            }
+            return [];
+        });
+
+        const tableLabelHeader = computed(() => {
```

Comment:
Does the corresponding table filter not have a name that could be used here?

Reply:
ah so the reason this is hardcoded in is because we would have to have a filter config for "Area" to get that header however for "Sex by Area" the filter is `area_level` which means if we try to get the header in a programmatic way, we would end up with a header called "Area Level"

Reply:
In the choropleth on filtering area level we call this filter "Detail", probably good to be consistent with that? Can we also translate the header name here

Reply:
for the choropleth, we have made a custom dropdown for it and hard coded in that we want these options so we called it detail, for the table tab, this is happening completely generically based on the metadata, i think if you change the label property in filters that you return with the preset for "Sex by Area" to "Detail" it will be consistent

Reply:
> which means if we try to get the header in a programmatic way, we would end up with a header called "Area Level"

Right, so we're filtering by `area_level` but the data we're showing in the column is for `area` - and we do have `area` filter elsewhere that we could pull the name from but we don't have one as part of this preset (and can't necessarily assume that it will always be present unless we make some other assumptions...)
I think I'd still prefer it to come in with the metadata somehow, but I guess this is fine for now. 

As Rob says, we do still need to translate though.